### PR TITLE
NDRCN-441 Join the "Can we contact you by email?" & "Enter your phone number" pages

### PR DESCRIPTION
**Pull request by shiromi**

### CODE REVIEW

File: app/utils/CheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -271,36 +271,24 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
...
 
-  def phoneNumber: Option[AnswerRow] = userAnswers.get(PhoneNumberPage) map {
+  def phoneNumber: Option[AnswerRow] = userAnswers.get(EmailAddressAndPhoneNumberPage) map {
     x =>
       AnswerRow(
         HtmlFormat.escape(messages("phoneNumber.checkYourAnswersLabel")),
-        HtmlFormat.escape(x),
-        Some(routes.PhoneNumberController.onPageLoad(CheckMode).url)
+        HtmlFormat.escape(x.phone.getOrElse("")),
+        Some(routes.EmailAddressAndPhoneNumberController.onPageLoad(NormalMode).url)
```
Comment (lmugnai) [inline app/utils/CheckYourAnswersHelper.scala:279]: Should be passing CheckMode here

File: app/utils/CheckYourAnswersHelper.scala [hunk 2]
```diff
@@ -271,36 +271,24 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
...
 
-  def emailAddress: Option[AnswerRow] = userAnswers.get(EmailAddressPage) map {
+  def emailAddress: Option[AnswerRow] = userAnswers.get(EmailAddressAndPhoneNumberPage) map {
     x =>
       AnswerRow(
         HtmlFormat.escape(messages("emailAddress.checkYourAnswersLabel")),
-        HtmlFormat.escape(x),
-        Some(routes.EmailAddressController.onPageLoad(CheckMode).url)
+        HtmlFormat.escape(x.email.getOrElse("")),
+        Some(routes.EmailAddressAndPhoneNumberController.onPageLoad(NormalMode).url)
```
Comment (lmugnai) [inline app/utils/CheckYourAnswersHelper.scala:288]: Should be passing CheckMode here

File: app/utils/CheckYourAnswersHelper.scala (hunk 1 above)
Comment (shiromi) [reply app/utils/CheckYourAnswersHelper.scala]: Fixed

File: app/utils/CheckYourAnswersHelper.scala (hunk 2 above)
Comment (shiromi) [reply app/utils/CheckYourAnswersHelper.scala]: Fixed

**Pull request merged**